### PR TITLE
Add the option to use aspera to download from ENA

### LINKED ISSUE #58: Support download tool: aspera
Provide aspera dowload scripts as an alternative to curl for zoooooooooooooooom when downloading

### PR DESCRIPTION
**Pull request by alexhbnr**
Adds the additional option "aspera" to the Streamlit sidebar and allows to download a Bash script that contains the commands to download the FastQ files from ENA using aspera instead of curl using the recommended commands following https://ena-docs.readthedocs.io/en/latest/retrieval/file-download.html#using-aspera

Since aspera requires the path to its installation location to load the config, this path is hidden using the Bash variable `ASPERA_PATH` that has to be set prior to executing the Bash script.

Would close #58.

### CODE REVIEW

**Review by jfy133 — CHANGES_REQUESTED** [review]
Missing soft-warning  that selecting aspera requires `aspera` to be installed, and ports _may_ need to be modified in the script depending on installation/firewall.

**Comment by alexhbnr** [comment]
Where would you put the soft warnings? I would put them into the documentary on readthedocs, which I haven't added it to yet. Or do you think it's necessary to state into in the Streamlit app?

**Comment by jfy133** [comment]
> Where would you put the soft warnings? I would put them into the documentary on readthedocs, which I haven't added it to yet. Or do you think it's necessary to state into in the Streamlit app?

Good point, yes the docs will be sufficient I think.

File: AMDirT/filter/streamlit.py [hunk 1]
```diff
@@ -61,15 +61,17 @@ def parse_args():
...
+    st.session_state.tag_name = st.selectbox(
+        label="Select an AncientMetagenomeDir release", options=tags
+    )
     options = ["No table selected"] + list(samples.keys())
-    st.session_state.table_name = st.selectbox(label="", options=options)
-    st.write(f"Only {' and '.join(supported_archives)} archives are supported for now")
-    st.write("## Select a download methods")
+    st.session_state.table_name = st.selectbox(label="Select a table", options=options)
+    st.session_state.height = st.selectbox('Number of rows to display', (10, 20,50, 100, 200), index=2)
     st.session_state.dl_method = st.selectbox(
```
Comment (maxibor) [inline AMDirT/filter/streamlit.py:70]:
```suggestion
    st.session_state.dl_method = st.selectbox(
            label="", options=["curl", "nf-core/fetchngs", "aspera"]
    )
    if st.session_state.dl_method == "aspera":
        st.warning(
            "You will need to set the `${ASPERA_PATH}` environment variable. See [documentation](https://amdirt.readthedocs.io) for more information."
        )
```

**Review by maxibor — APPROVED** [review]

**Comment by maxibor** [comment]
I suggest to include the aspera doc in a separate PR.
With the added warning about the aspera variable, this PR looks good to me.

**Pull request merged**